code/modules/research/anomaly/raw_anomaly.dm

```diff
@@ -68,6 +68,12 @@
 	anomaly_type = /obj/item/assembly/signaler/anomaly/dimensional
 	icon_state = "rawcore_dimensional"
 
+/obj/item/raw_anomaly_core/ectoplasm //Has no cargo order option, but can sometimes be a roundstart pick
+	name = "\improper ectoplasm anomaly core"
+	desc = "The raw core of an ectoplasmic anomaly. It wants to share it's secrets with you."
```

Comment:
```suggestion
	desc = "The raw core of an ectoplasmic anomaly. It wants to share its secrets with you."
```

I think this works better though:

```suggestion
	desc = "The raw core of an ectoplasmic anomaly. It wants to share secrets with you."
```

Because ghosts can see a lot of secret things.... Just do one of the two and we're solid.